redaxo/src/addons/metainfo/lang/de_de.lang

```diff
@@ -37,6 +37,8 @@ minfo_field_params_notice_6 = Beispiel:<br /> <code>category="1" types="gif,jpg"
 minfo_field_params_notice_7 = Beispiel:<br /> <code>category="1" types="gif,jpg" preview="1"</code>
 minfo_field_params_notice_8 = Beispiel:<br /> <code>category="1"</code>
 minfo_field_params_notice_9 = Beispiel:<br /> <code>category="1"</code>
+minfo_field_params_notice_10 = Beispiel:<br /> <code>start="1990" end="2030"</code>
+minfo_field_params_notice_11 = Beispiel:<br /> <code>start="1990" end="2030"</code>
```

Comment:
Sind die beispiele in dee UI noch intuitiv/eindeutig für den user, oder sollte man bei manchen erwähnen dass sie nur mit bestimmten input-types funktionieren?

Reply:
Die werden bereits getogglet. 10 = date, 11 = datetime